pkg/ostree/schedule_backup_test.go

```diff
@@ -0,0 +1,313 @@
+package ostree
+
+import (
+	"fmt"
+	"testing"
+)
+
+var ostreeStatusJSON = `{
```

Comment:
Can we avoid using "fedora" examples here? 
It is not a supported OS and it may create a different impression / confusion.